index.html

```diff
@@ -0,0 +1,221 @@
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard</p>
+                    <ul>
+                        <li>Html</li>
+                        <li>Boostrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+                <div class="context-projects-3">
+                    <span>Data Dashboard Healthcare</span>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard</p>
+                    <ul>
+                        <li>Html</li>
+                        <li>Boostrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+                <div class="context-projects-3">
+                    <span>Data Dashboard Healthcare</span>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard</p>
+                    <ul>
+                        <li>Html</li>
+                        <li>Boostrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+            </div>
+
+        </section>
+        <section class="section-about">
+            <div class="about-text">
+                <h2>About me</h2>
+                <p>
+                    Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+                <button class="about-button" type="button">
+                    Get My Resume
+                </button>
+                <div class="about-background"></div>
+            </div>
+            <hr>
+            <div class="about-language">
+                <ul>
+                    <li>Languages
+                        <div class="sublist">
+                            <span class="sublist-title">Javascript</span>
+                            <span class="sublist-title">Ruby</span>
+                            <span class="sublist-title">Html</span>
+                            <span class="sublist-title">Css</span>
+                        </div>
+                    </li>
+                    <li>Frameworks
+                        <div class="sublist">
+                            <span class="sublist-title">Bootstrap</span>
+                            <span class="sublist-title">Ruby on Rails</span>
+                            <span class="sublist-title">RSPec</span>
+                            <span class="sublist-title">CapyBara</span>
+                            <span class="sublist-title">Selenium</span>
+                        </div>
```

Comment:
Please use the shapes for Headings in the technology section from the design page. Moreover please use small font for lists. KIndly see the screenshot below for reference.
![tech](https://user-images.githubusercontent.com/35504149/135471669-b8f3e23d-02b2-47eb-bfa7-3877c985db7a.png)